lib/actions/usbsdfuTargetActions.js

```diff
@@ -182,75 +252,24 @@ export function canWrite() {
             RegionName.APPLICATION,
             RegionName.SOFTDEVICE,
             RegionName.BOOTLOADER,
-            RegionName.MBR_OR_APP,
         ]);
-        let fileRegionNames = new Set();
+        let detectedRegionNames = new Set();
         file.regions.forEach(r => {
             if (r.name && regionChecklist.includes(r.name)) {
-                fileRegionNames = fileRegionNames.add(r.name);
+                detectedRegionNames = detectedRegionNames.add(r.name);
             }
         });
         let isWritable = false;
 
-        switch (fileRegionNames.size) {
-            case 0: {
-                dispatch(targetActions.targetWarningKnownAction('No writable region is detected'));
-                break;
-            }
-            case 1: {
-                switch (fileRegionNames.first()) {
-                    case RegionName.APPLICATION: {
-                        // TODO: create user input for firmware version
-                        const fwVersion = 1;
-                        dispatch(targetActions.initPacketFwVersionKnownAction(
```

Comment:
I suspect that by removing these `initPacket...Action()`s  they might not be used anymore. In that case please follow what needs to be cleaned, since unused exports are not reported by linter. `INIT_PACKET_...` types, ids, action creators and maybe even states can be removed or simplified.

Reply:
Removed